console/local.go

```diff
@@ -17,16 +17,16 @@ import (
 func Server(ctx context.Context) (http.Handler, error) {
 	logger := log.FromContext(ctx)
 	logger.Infof("Starting console dev server")
-	output := logger.WriterAt(log.Info)
```

Comment:
I think sending it to `Trace` or `Debug` would be better, otherwise it will be impossible to debug.